### ISSUE TITLE
datasets w/out licenses

### ISSUE THREAD

**Issue by keeganmcbride** [okestonia/jkan#75]
[1] "https://opendata.riik.ee/datasets/andmestik-rahvastikuregistris-leiduvate-nimedega/"            
 [2] "https://opendata.riik.ee/datasets/digar/"                                                       
 [3] "https://opendata.riik.ee/datasets/harku-vallavalitsus/"                                         
 [4] "https://opendata.riik.ee/datasets/muuseumide-infos-steemi-avaandmed/"                           
 [5] "https://opendata.riik.ee/datasets/tallinn-arvudes/"                                             
 [6] "https://opendata.riik.ee/datasets/tallinn-avalike-urituste-load/"                               
 [7] "https://opendata.riik.ee/datasets/tallinn-ilutulestike-load/"                                   
 [8] "https://opendata.riik.ee/datasets/tallinn-linnaosade-trukikaardid/"                             
 [9] "https://opendata.riik.ee/datasets/tallinn-raieload/"                                            
[10] "https://opendata.riik.ee/datasets/tallinn-services/"                                            
[11] "https://opendata.riik.ee/datasets/tallinn-spordiurituste-load/"                                 
[12] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-arhiiv/"                         
[13] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-avalik-kord/"                    
[14] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-ettevotlus-kaubandus-ja-reklaam/"
[15] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-haridus/"                        
[16] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-heakord-ja-haljastus/"           
[17] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-keskkonnakaitse/"                
[18] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-kultuur/"                        
[19] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-linnajuhtimine/"                 
[20] "https://opendata.riik.ee/datasets/tallinn-teenuste-kirjeldused-muinsuskaitse-ja-miljooalad/"    
[21] "https://opendata.riik.ee/datasets/tallinna-andmekogud/"                                         
[22] "https://opendata.riik.ee/datasets/tallinna-eelarve/"                                            
[23] "https://opendata.riik.ee/datasets/tallinna-geoportaal/"                                         
[24] "https://opendata.riik.ee/datasets/tallinna-planeeringute-register-tpr/"                         
[25] "https://opendata.riik.ee/datasets/tallinna-teenused/"                                           
[26] "https://opendata.riik.ee/datasets/valimiste-avaandmed/"

**Comment by infokujur** [comment]
I'll reach out.

**Comment by keeganmcbride** [comment]
@otuxe @infokujur  Did we agree on a default license to use?

**Comment by otuxe** [comment]
We didn't agree on the default license. However, we should tell in the terms and conditions of open data portal, that if the organization has not issued a licence, the portal assigns a "default" licence. For now we can use Creative Commons.. .

**Comment by keeganmcbride** [comment]
Okay, I will add CC3.0 then?

**Comment by otuxe** [comment]
Yes! But also add T&C to the portal and mentioned the default license part.

**Comment by infokujur** [comment]
@otuxe - kas valitsusveebidele on ka mingisugused standartsed veebikasutustingimused välja töötatud. Lihtne googeldamine ei aitanud.

**Comment by otuxe** [comment]
> @otuxe - kas valitsusveebidele on ka mingisugused standartsed veebikasutustingimused välja töötatud. Lihtne googeldamine ei aitanud.

Ei ole standartseid kasutustingimusi. Suhtlesin selles osas Riigikantseleiga. Võiksime aluseks võtta mõne riigiasutuse poolt kasutatavad kasutustingimused.

**Comment by infokujur** [comment]
Kas Riigikantseleis soovitas mõnda riigiasutust, kelle kasutustingimusi võtta kasutusele?

**Comment by otuxe** [comment]
Otseselt mitte, aga ma vaataks RIKi kasutustingimusi näitena: https://www.rik.ee/et/asutusest/kasutustingimused

**Comment by infokujur** [comment]
See on vist kõikide teenuste kohta, mitte ühe veebilehe kasutustingimused. Või soovid kuidagi midagi laiemalt raamistada?
Teil jurist kindlasti oskab siin paremini soovitada.

Kui üldosa paigas, siis saame enda poolt avaandmete koha kirja panna.

Ja siis peale seda on teil vaja ka maja sees need käskkirjaga? kinnitada?

**Comment by otuxe** [comment]
Tegu on ikkagi e-teenuste kasutustingimustega (RIK). Meie jurist ei oska antud teemal kaasa rääkida, pole pädev sellel teemal. Riigikantselei pole ka antud teemasse süüvinud. Ühesõnaga, meil tuleb ise kasutustingimused luua. Palun tehke põhi Google docsi ja me vaatame selle üle. Käskkirjaga ei ole vaja kinnitada kasutustingimusi.
